.circleci/config.yml

```
@@ -120,7 +120,7 @@ jobs:
       - install_js
       - run:
           name: '`yarn prettier` changes committed?'
-          command: yarn prettier --check
+          command: yarn prettier:all --check
```

Otherwise it fails on the next step: https://app.circleci.com/pipelines/github/mui/mui-toolpad/10406/workflows/ed6e5232-f8d3-42e2-8f00-44e4e25b2375/jobs/45735

```suggestion
          command: |
            yarn prettier:all --check
            git add -A && git diff --exit-code --staged
```